src/drivers/net/dwc_gmac/dwc_gmac.c

```diff
@@ -0,0 +1,596 @@
+/**
+ * @file
+ *
+ * @date Sep 4, 2018
+ * @author Anton Bondarev
+ */
+#include <util/log.h>
+
+#include <errno.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <drivers/common/memory.h>
+
+#include <hal/reg.h>
+#include <kernel/irq.h>
+
+#include <net/inetdevice.h>
+#include <net/l0/net_entry.h>
+#include <net/l2/ethernet.h>
+#include <net/netdevice.h>
+#include <net/skbuff.h>
+#include <net/util/show_packet.h>
+#include <net/mii.h>
+#include <net/phy.h>
+
+#include "dwc_gmac.h"
+#include "dwc_desc.h"
+
+#include <embox/unit.h>
+#include <framework/mod/options.h>
+
+
+extern void dcache_inval(const void *p, size_t size);
+extern void dcache_flush(const void *p, size_t size);
+
+EMBOX_UNIT_INIT(dwc_init);
+
+#define BASE_ADDR  OPTION_GET(NUMBER, base_addr)
+#define IRQ_NUM    OPTION_GET(NUMBER, irq_num)
+
+#define RX_DESC_QUANTITY 0x20
+#define TX_DESC_QUANTITY 0x10
+
+struct dwc_priv {
+	uint32_t base_addr;
+
+	int phy_id;
+	uint32_t mii_clk;
+	uint8_t macaddr[ETH_ALEN];
+	struct sk_buff *rx_skb_pool[RX_DESC_QUANTITY];
+	void *rxdesc_ring_paddr;
```

Comment:
`rxdesc_ring_paddr` and `rxdesc_ring_paddr` are used only in `dwc_hw_init`, we can simply remove this fields.